apps/ledger-live-desktop/src/renderer/screens/exchange/Swap2/Form/SwapWebView.tsx

```diff
@@ -62,12 +62,9 @@ export type SwapWebProps = {
 
 export const SWAP_WEB_MANIFEST_ID = "swap-live-app-demo-0";
 
-const SwapWebAppWrapper = styled.div<{ isDevelopment: boolean }>(
-  ({ isDevelopment }) => `
-  ${!isDevelopment ? "height: 0px;" : ""}
+const SwapWebAppWrapper = styled.div<{ isDevelopment: boolean }>`
   width: 100%;
-`,
-);
+`
 
```

Comment:
Could we just remove the wrapper is there is no difference between dev and prod now?